### ISSUE TITLE
IKEA TRÅDFRI bulb E27 WS clear 806lm

### ISSUE THREAD

**Issue by bluemoehre** [dresden-elektronik/deconz-rest-plugin#8005]
### Is there already an existing issue for this?

- [X] I have searched the existing issues and there is none for my device

### Product name

TRÅDFRI bulb E27 WS clear 806lm

### Manufacturer

IKEA of Sweden

### Model identifier

TRADFRI bulb E27 WS clear 806lm

### Device type to add

Light

### Node info

![image](https://github.com/user-attachments/assets/5742d576-4aa2-4566-b8be-0364507352ac)

### Endpoints and clusters

![image](https://github.com/user-attachments/assets/5e507a87-27e8-4ee7-b463-c7f8888ce4ad)

### Basic

![image](https://github.com/user-attachments/assets/fc6ffc8c-aef0-446e-bd91-85e1471d34a2)

### On/Off

![image](https://github.com/user-attachments/assets/4815ed4b-ef42-4639-9bf6-480001bcecac)

### Level Control

![image](https://github.com/user-attachments/assets/e09c641c-5671-4227-bf7d-d62490beb604)

### Color Control

![image](https://github.com/user-attachments/assets/00b992db-2120-4475-a2d6-64df4f09e21f)
![image](https://github.com/user-attachments/assets/adb4bbd4-480a-4061-8694-b3da2b086d72)
![image](https://github.com/user-attachments/assets/802a97d7-174a-4f97-b20c-1ed4de08fc7e)

**Comment by mattreim** [comment]
And here change the “modelid”, “product” and the name of this json. Then it should work and the intervals should be checked for all IKEA DDF's.

https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/ikea/tradfri_bulb_e14_ws_opal_600lm.json

**Comment by bluemoehre** [comment]
> and the intervals should be checked for all IKEA DDF's.

You are right. Some attributes are polled every `5s`. That will cause spamming the network, am I wrong? Seems to be a larger issue and related to a bunch of devices. Therefore that topic should have a dedicated ticket.

**Comment by mattreim** [comment]
It's true, but what's more important is that it works.

**Comment by ebaauw** [comment]
I think all IKEA devices I’ve seen support attribute reporting.  Better use that instead of polling.

**Comment by bluemoehre** [comment]
Waiting for other PRs to get merged before continuing here.

**Comment by bluemoehre** [comment]
bump

**Comment by bluemoehre** [comment]
bump

**Comment by bluemoehre** [comment]
bump

**Comment by mattreim** [comment]
Is merged [tradfri_bulb_e27_ws_clear_806lm.json](https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/ikea/tradfri_bulb_e27_ws_clear_806lm.json) and the issue can be closed.
